addon/helpers/cat-color-scale.js

```diff
@@ -1,39 +1,36 @@
 import Ember from 'ember';
 import {
-  scaleCategory10,
-  scaleCategory20b,
-  scaleCategory20c,
-  scaleCategory20
+  scaleOrdinal,
+  schemeCategory10,
+  schemeCategory20b,
+  schemeCategory20c,
+  schemeCategory20
 } from 'd3-scale';
-import guidDomainScale from '../utils/guid-domain-scale';
+
+// import guidDomainScale from '../utils/guid-domain-scale';
 const {
-  get,
   assert,
-  isEmpty
+  isPresent,
 } = Ember;
 const SCALES = {
-  scaleCategory10,
-  scaleCategory20b,
-  scaleCategory20c,
-  scaleCategory20
-};
-
-const WHITELIST = {
-  '10': true,
-  '20': true,
-  '20b': true,
-  '20c': true
+  10: schemeCategory10,
+  '20b': schemeCategory20b,
+  '20c': schemeCategory20c,
+  20: schemeCategory20,
 };
 
 export function catColorScale([type, domain]) {
   let capType = type.toString().toLowerCase();
-  assert(`${type} is not a valid sequential color scale name`, capType in WHITELIST);
 
-  let catScale = get(SCALES, `scaleCategory${capType}`);
-  let scale = guidDomainScale(catScale());
+  let catScale = SCALES[capType];
+  assert(`${type} is not a valid sequential color scale name`, !!catScale);
+
+  // let scale = guidDomainScale(catScale);
```

Comment:
My only comment was in regards to this line — I think for sake of consistency, either this `guidDomainScale` is included in all ordinal type scales or not at all. (It's used in a few other places). 

For reference, what this does is adds support for using arrays of any type, rather than just arrays of strings/numbers as the domain.

So you can do this:

```js
let a = {};
let b = {};
let c = {};
scale.domain([ a, b, c ]).range([0, 100]);

scale(a); // => 0
scale(b); // => 50
scale(c); // => 100
```

If we leave this removed, then the behavior would be something like:

```js
let a = {};
let b = {};
let c = {};
scale.domain([ a, b, c ]).range([0, 100]);

scale(a); // => 0
scale(b); // => 0
scale(c); // => 0
```

...since d3 does a `.toString()` on each of the items in a domain to generate its key. `guidDomainScale` overwrites that behavior to use `Ember.guidFor()`.